### PR TITLE
Fotter update 1

### PR DESCRIPTION
**Pull request by karczujr**

### CODE REVIEW

File: src/components/Footer.astro [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    </div>
+  </section>
+</footer>
+
+<style lang="scss">
+  footer {
+    margin-top: auto;
+    border-top: 1px solid var(--footer-color);
+
+    .wrapper {
```
Comment (ArturBa) [inline src/components/Footer.astro]: .wrapper === section

File: src/components/Footer.astro [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+      gap: 16px;
+      display: block;
+    }
+
+    p {
+      font-size: 18px;
+      font-weight: 400;
+      line-height: 1.5;
+      color: var(--footer-color);
+    }
```
Comment (ArturBa) [inline src/components/Footer.astro]: this looks like `.small-body` from `_typography.scss`

File: src/components/Footer.astro [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+    img {
+      height: 14px;
```
Comment (ArturBa) [inline src/components/Footer.astro:39]: don't we use 4/8 pt grid? 14 isn't in that one

File: src/components/Footer.astro [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+      font-weight: 400;
+
+      &::after {
+        background-color: var(--black);
+      }
+
+      &:hover {
+        color: var(--black);
+      }
+    }
```
Comment (ArturBa) [inline src/components/Footer.astro]: this might be simplified after merging #8

File: src/components/Footer.astro [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+          >Unicorn Studio</a
+        >
+      </p>
+    </div>
+  </section>
+</footer>
+
+<style lang="scss">
+  footer {
+    margin-top: auto;
```
Comment (ArturBa) [inline src/components/Footer.astro:21]: this might cause some issues in the future.

Comment (ArturBa) [reply src/components/Footer.astro:21]: keep that in mind for if you encourage any issues

**Review by ArturBa — APPROVED** [review]

File: src/components/Footer.astro (hunk 2 above)
Comment (karczujr) [reply src/components/Footer.astro]: Look at my combo

Comment (karczujr) [reply src/components/Footer.astro]: Is it enough simply and clean? Good that you do not have a chance how it had looked before

File: src/components/Footer.astro (hunk 4 above)
Comment (karczujr) [reply src/components/Footer.astro]: I did it.

File: styles/sass/base/_typography.scss [hunk 6]
```diff
@@ -3,29 +3,29 @@ h1 {
...
+  color: var(--headline-color);
 }
 
 .large-body {
   font-size: 24px;
   font-weight: 400;
   line-height: 1.5;
   padding-bottom: 40px;
-  color: #4a4a4a;
+  color: var(--paragraph-color);
```
Comment (ArturBa) [inline styles/sass/base/_typography.scss:22]: 💜

File: src/components/Footer.astro [hunk 7]
```diff
@@ -0,0 +1,68 @@
+<footer>
+  <section class="small-body">
```
Comment (ArturBa) [inline src/components/Footer.astro:2]: nice!

File: src/components/Footer.astro [hunk 8]
```diff
@@ -0,0 +1,68 @@
...
+      align-items: center;
+    }
+
+    .other {
+      display: flex;
+      justify-content: space-between;
+    }
+
+    a {
+      --footer-link-color: var(--black);
```
Comment (ArturBa) [inline src/components/Footer.astro]: you should define `--link-color` instead of `--footer-link-color`

File: src/components/Footer.astro [hunk 9]
```diff
@@ -0,0 +1,68 @@
...
+
+      color: var(--footer-link-color);
```
Comment (ArturBa) [inline src/components/Footer.astro]: this might be removed after you declare the proper variable

File: src/components/Footer.astro [hunk 10]
```diff
@@ -0,0 +1,68 @@
...
+      font-weight: 400;
+
+      &::after {
+        background-color: var(--footer-link-color);
```
Comment (ArturBa) [inline src/components/Footer.astro]: this might be removed after settin proper variable

File: src/components/Footer.astro [hunk 11]
```diff
@@ -0,0 +1,68 @@
...
+      }
+
+      &:hover {
+        color: var(--footer-link-color);
```
Comment (ArturBa) [inline src/components/Footer.astro]: this might be removed, but to be confirm if the variable `--link-color` shoudnt' be redefined in here

**Pull request merged**